backend/README.md

```diff
@@ -9,31 +9,18 @@
 
 ## Developement
 
-Install and run the backend using `npm install`, then `npm run app`, or simply run `npm run backend` from the project root folder.
+Install and run the backend using `docker compose up`  
+If you change the fixtures afterwards you have to run `docker compose up --build` to reload the new fixtures to the image.
 
 ### Environment variables
```

Comment:
Is it possible to run `docker compose up` without the `.env` file? If not, we should rearrange the README so requirements comes before starting the app. 😊

Also, is there any situation where one wouldn't want to run using Docker? As in, any reason to keep both installation instructions?